GameData/RP-1/Contracts/Commercial Applications 2/FirstTargetedMolniya.cfg

```diff
@@ -0,0 +1,169 @@
+CONTRACT_TYPE
+{
+	name = FirstTargetedMolniya
+	title = First Targeted Molniya Orbit Satellite
+	group = CommApp2
+	agent = Federation Aeronautique Internationale
+
+	description = <b>Program: Advanced Commercial Applications<br><b>Type: <color=blue>Optional</color></b><br><br>A Molniya orbit is a type of highly elliptical orbit with an inclination of 63.4 degrees, an argument of perigee in the very high or low latitudes, and an orbital period of one half of a sidereal day. This causes it to spend most of its time over two areas on the ground at opposing longitudes. Place a commercial satellite into a Molniya orbit so that it loiters over the specified ground location for at least 6 consecutive hours.<br><br>Historical example: Molniya 1-1 (1,600kg, Molniya)<br><br><b><color=yellow>NOTE: The satellite will be destroyed upon completion of the contract. This simulates transfer of the payload back to the customer.</color>
+
+	synopsis = Launch a commercial satellite into a Molniya orbit
+
+	completedMessage = Success! The satellite is in orbit to loiter over the desired area.
+
+	sortKey = 501
+
+	cancellable = true
+	declinable = false
+	autoAccept = false
+	minExpiry = 0
+	maxExpiry = 0
+	maxCompletions = 1
+	maxSimultaneous = 1
+	deadline = 0
+
+	targetBody = HomeWorld()
+
+
+	// ************ REWARDS ************
+	prestige = Trivial       // 1.0x
+	advanceFunds = 0
+	rewardScience = 0
+	rewardFunds = 0
+	failureFunds = 0
+	rewardReputation = 60
+	failureReputation = 0 // was @rewardReputation
+
+
+
+	// ************ REQUIREMENTS ************
+
+	REQUIREMENT
+	{
+		name = ProgramActive
+		type = ProgramActive
+		program = CommercialApplications2
+	}
+
+	BEHAVIOUR
+	{
+		name = WaypointGenerator
+		type = WaypointGenerator
+
+		WAYPOINT
+		{
+			name = Molniya Orbit Target
+			icon = thermometer
+			altitude = 0
+			latitude = @/targetLat
+        	longitude = @/targetLon
```

Comment:
Mixed tabs and spaces here and in a few other spots. Should be easy enough to spot in the web GUI.